File: src/main/java/io/github/trcdevelopers/clayium/mixins/ClayiumLateMixinLoadingPlugin.java
package io.github.trcdevelopers.clayium.mixins;

import zone.rong.mixinbooter.ILateMixinLoader;

import java.util.Collections;
import java.util.List;

@SuppressWarnings("unused")
public class ClayiumLateMixinLoadingPlugin implements ILateMixinLoader {
    @Override
    public List<String> getMixinConfigs() {
        return Collections.singletonList("mixins.clayium.json");
    }
}

File: src/main/kotlin/io/github/trcdevelopers/clayium/core/ClayiumLateMixinLoadingPlugin.kt
package io.github.trcdevelopers.clayium.core

import zone.rong.mixinbooter.ILateMixinLoader

@Suppress("unused")
class ClayiumLateMixinLoadingPlugin : ILateMixinLoader {
    override fun getMixinConfigs(): MutableList<String?> {
        return mutableListOf("mixins.clayium.json")
    }
}
